src/components/AddBookForm.js

```diff
@@ -0,0 +1,14 @@
+export const AddBookForm = () => (
+  <div>
+    <h2>ADD NEW BOOK</h2>
+    <form onSubmit="#">
```

Comment:
- [ ] Our form is missing an input element for the books' `author`. Kindly fix this.

Reply:
Based on the design from [Zeplin](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) the author field wasn't part of it. But I have added it as suggested.